build.gradle

```diff
@@ -149,6 +149,7 @@ swaggerSources {
 			rawOptions = [
 					"--api-package", "${artifactGroup}.generated.controller",
 					"--model-package", "${artifactGroup}.generated.model",
+					"--model-name-prefix", "Api",
```

Comment:
Did you do the trick where you refactor the generated class in IJ first to fix up all the references to it and then generate them with the prefix?

Reply:
If only :(